lib/Storage/Path.cpp

```diff
@@ -0,0 +1,32 @@
+/// \file S3StorageClient.cpp
+/// \brief Implementation of StorageClient operations in S3
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "revng/Storage/LocalStorageClient.h"
+#include "revng/Storage/Path.h"
+#include "revng/Storage/StorageClient.h"
+
+namespace revng {
+
+namespace {
```

Comment:
Uhm, I'd prefer anonymous namespaces to be top-level namespaces.

Reply:
Moved